exporter/sentryexporter/sentry_exporter.go

```diff
@@ -192,7 +229,26 @@ func convertToSentrySpan(span pdata.Span) (sentrySpan *SentrySpan) {
 		StartTimestamp: UnixNanoToTime(span.StartTime()),
 		EndTimestamp:   UnixNanoToTime(span.EndTime()),
 		Status:         status,
+		ResourceTags:   resourceTags,
+	}
+
+	if !library.IsNil() {
+		name := library.Name()
+		version := library.Version()
+
+		if name != "" && version != "" {
+			sentrySpan.LibName = name
+			sentrySpan.LibVersion = version
+		} else {
+			sentrySpan.LibName = "otel.sentry.exporter"
+			sentrySpan.LibVersion = otelSentryExporterVersion
+		}
+	} else {
+		sentrySpan.LibName = "otel.sentry.exporter"
```

Comment:
What about we name it?
```suggestion
		sentrySpan.LibName = "sentry.opentelemetry.exporter"
```
We're not shy of long names for SDK. i.e: `sentry.javascript.browser`

Reply:
+1 on starting with `sentry`. The full format is:
`entity.ecosystem[.flavor]` (documented [here](https://develop.sentry.dev/sdk/event-payloads/sdk/#attributes)). Since this is not a flavor of an existing SDK, I guess `sentry.opentelemetry` would already be enough.

Reply:
I suggest to append collector since individual sdks can also exporters.